src/pages/account/organization/new/form/fields/IncumbencyCertOption.component.js

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+
+import { useFormikContext } from 'formik';
+import { Upload } from '@tidb-community/ui';
+import { Form } from 'formik-antd';
+import data from '../form.data';
+import { api } from '@tidb-community/datasource';
+
+const { incumbencyCert } = data.form.verificationType;
+
+// uploadIncumbencyCert 不传参数代表删除文件
```

Comment:
Please use English comment as the repo is open-source.

Reply:
This comment is outdated and should be deleted.